File: exercises/small_problems/easy2/retire.rb
puts "How old are you?"
age = gets.chomp.to_i

puts "How old do you want to be when you retire?"
retire_age = gets.chomp.to_i

time_left = retire_age - age

this_year = Time.now.year

puts "It's #{this_year} now. You will retire in #{this_year + time_left}."
puts "You only have #{time_left} years until you retire!"

File: exercises/jsSmallProblems/easy3/retire.js
const currentAge = prompt('What is your age?');
const retireAge = prompt('At what age would you like to retire?');

const yearsLeft = retireAge - currentAge;

const currentYear = Date().getFullYear();

const retireYear = currentYear + yearsLeft;

console.log(`It's ${currentYear}. You will retire in ${retireYear}.`);
console.log(`You have only ${yearsLeft} years of work to go!`);
